### PR TITLE
Polkadot Protocol > Protocol Components > Overview

### PR DESCRIPTION
**Pull request by kapetan3sid**
Migrated https://wiki.polkadot.network/docs/build-protocol-info#existential-deposit page

### CODE REVIEW

**Comment by kapetan3sid** [comment]
@0xLucca @CrackTheCode016 I think we should remove **Tokens** and **FAQ** sections from this page.

It feels to me that they don't belong here, thoughts?

File: polkadot-protocol/protocol-components/overview.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+### Redomination
+
+Polkadot conducted a community poll, which ended on **27 July 2020** at block **888,888** to decide whether to redenominate the DOT token. 
+
+The stakeholders chose to redenominate the token, changing the value of 1 DOT from 1e12 plancks to 1e10 plancks. 
+
+Importantly, this did not affect the total number of base units (plancks) in the network, only how a single DOT is represented.
+
+The redenomination became effective 72 hours after transfers were enabled, occurring at block 1,248,326 on 21 August 2020 around 16:50 UTC.
```
Comment (nhussein11) [inline polkadot-protocol/protocol-components/overview.md:29]: According to the wiki: https://wiki.polkadot.network/docs/learn/learn-redenomination, this redenomination thing has happened  at block 1_248_328

```suggestion
The redenomination became effective 72 hours after transfers were enabled, occurring at block 1,248,328 on 21 August 2020 around 16:50 UTC.
```

File: polkadot-protocol/protocol-components/overview.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+## Addresses
+
+In Polkadot and most Substrate-based chains, user accounts are represented by a **32-byte (256-bit)** `AccountId`, which is typically — but not always — the public key of a cryptographic key pair.
```
Comment (nhussein11) [inline polkadot-protocol/protocol-components/overview.md:33]: We're switching from Substrate to Polkadot SDK when possible
```suggestion
In Polkadot and most Polkadot SDK-based chains, user accounts are represented by a **32-byte (256-bit)** `AccountId`, which is typically — but not always — the public key of a cryptographic key pair.
```

File: polkadot-protocol/protocol-components/overview.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+It shares characteristics with Bitcoin's `Base58Check` encoding, such as a version prefix, hash-based checksum, and base-58 encoding, ensuring addresses are network-specific and resistant to errors.
+
+!!!note
+    For detailed encoding methods and a comprehensive list of network prefixes, refer to the [`SS58-registry`](https://github.com/paritytech/ss58-registry){target=\_blank} repository 
+
+!!!warning "Do not use regular expressions (regex) to validate addresses"
+    Always verify an address using its prefix and checksum to ensure its validity. 
+    
+    The [**Substrate API Sidecar**](https://github.com/paritytech/substrate-api-sidecar){target=\_blank} offers an endpoint at accounts/{accountId}/validate, which returns a boolean isValid response for any provided address.
```
Comment (nhussein11) [inline polkadot-protocol/protocol-components/overview.md:45]: Maybe we can also add a link to that endpoint in the API sidecar docs

```suggestion
    The [**Substrate API Sidecar**](https://github.com/paritytech/substrate-api-sidecar){target=\_blank} offers an endpoint at [accounts/{accountId}/validate](https://paritytech.github.io/substrate-api-sidecar/dist/#:~:text=for%20an%20account.-,GET,-/accounts/%7Baddress%7D){target=\_blank}, which returns a boolean `isValid` response for any provided address.
```

File: polkadot-protocol/protocol-components/overview.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+
+Polkadot supports several [cryptographic](link to cryptography page on our site) key pairs and signing algorithms to enhance security and versatility in transactions. 
+
+The primary types include:
+
+- **Ed25519**: This is a popular signature scheme known for its speed and security, widely used in various blockchain applications.
+
+- **Sr25519**: This utilizes Schnorr signatures on the Ristretto group, providing extra security features, including resistance to certain types of attacks that can affect other signature schemes.
+
+- **ECDSA**: Specifically, Polkadot employs ECDSA signatures on the `secp256k1` curve, a standard in many cryptocurrencies like Bitcoin.
```
Comment (nhussein11) [inline polkadot-protocol/protocol-components/overview.md:57]: Just formatting details here...
```suggestion
- `Ed25519` - this is a popular signature scheme known for its speed and security, widely used in various blockchain applications

- `Sr25519` - this utilizes Schnorr signatures on the Ristretto group, providing extra security features, including resistance to certain attacks that can affect other signature schemes

- `ECDSA` - specifically, Polkadot employs ECDSA signatures on the `secp256k1` curve, a standard in many cryptocurrencies like Bitcoin
```

File: polkadot-protocol/protocol-components/overview.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+
+This concept aligns with how transactions function in Ethereum and Bitcoin, where signed transactions are essential for validating and processing operations on the blockchain.
+
+#### Unsigned Transactions
+
+Some transactions require unsigned transactions because they cannot be initiated by an account with sufficient funds to pay fees. 
+
+For instance, when a user claims DOT tokens from an Ethereum DOT indicator contract to a new DOT address, the new address starts with zero funds and cannot cover transaction fees. 
+
+Thus, unsigned transactions are essential in enabling actions in such scenarios, ensuring users can participate without funding limitations.
```
Comment (nhussein11) [inline polkadot-protocol/protocol-components/overview.md:134]: This entire section overlaps with: https://github.com/polkadot-developers/polkadot-docs/pull/77/files#diff-0ab151ab7f6f1f93b152f9f43576a07577693fb69d7f117072556994daed5c17R12

I'd leave only the extrinsic paragraph here and then maybe add a link to the docs of transactions for further details.

**Comment by nhussein11** [comment]
> @kapetan3sid Upon looking at this PR, I have the following thoughts:
> 
> Most of the content here can be extrapolated into other pages, for this to be an "Overview", imho, it couldn't contain such fine details of things like transaction mortality (which should probably go into the transactions page).
> 
> Some of the information is redundant as well, for example, explaining signed/unsigned/inhereent transactions here, and also explaining it elsewhere. We can include these as concepts in the overview, but ultimately point towards a dedicated page for transactions, so we don't have to maintain more than one place.
> 
> This overview page could contain a **brief** summary of the following:
> 
> * What the Polkadot relay chain does, the base currency, whereon we can link to details on DOT
> * Explain the concept of rollups/parachains, collators and validators, and then link to those pages
> * A blip about hybrid consensus, and link to that page for more detail
> * A blip about forkless upgrades, governance, and how they work together
> * A general extrinsic section, explaining how they are different from transactions in their identification (already in the document)
> * Encoding section can stay, but maybe we beef it up a little bit and link it to the other page for SCALE encoding
> * Fees section can stay
> * Events section can go IMHO, maybe they can be mentioned in the extrinsic section and linked away.
> * Link to the Polkadot Spec, which can also be used as a point of reference: https://spec.polkadot.network/id-polkadot-protocol
> 
> My thoughts so far, curious to hear others!

Oh, I read this just after providing my suggestions, sorry!

I agree that those items should be included here, as long as they are explained briefly, without diving too much into details. For details or further explanations about implementations, we can link that up to dedicated pages (as it's the case with transactions basics)

**Review by 0xLucca — COMMENTED** [review]
I agree with what the guys said before. Most of the content of this page seems duplicated from the pages we are writing for each specific section ([Cryptography](https://github.com/polkadot-developers/polkadot-docs/pull/74), [Address Formats](https://github.com/polkadot-developers/polkadot-docs/pull/49), etc). We can probably wait until those pages are merged, and based on that populate this section with the minimum amount of information and linking those pages. What do you think?

**Comment by kapetan3sid** [comment]
@CrackTheCode016 @0xLucca @nhussein11 I'll wait until it's decided what to do with this page before I jump addressing your comments

**Comment by kapetan3sid** [comment]
Closing this PR - this content will exist on another section

**Pull request closed without merging**